validation_testing/cdk_federation_infra_provisioning/app/package.json

```diff
@@ -13,15 +13,15 @@
   "devDependencies": {
     "@types/jest": "^29.4.0",
     "@types/node": "18.11.18",
-    "aws-cdk": "2.63.2",
+    "aws-cdk": "2.118.0",
     "jest": "^29.4.1",
     "ts-jest": "^29.0.5",
     "ts-node": "^10.9.1"
   },
   "dependencies": {
     "@aws-cdk/aws-glue-alpha": "2.63.2-alpha.0",
```

Comment:
can you also upgrade this version to keep it the same as the normal library; and test please.